test/SwooleRequestHandlerRunnerTest.php

```diff
@@ -63,17 +53,20 @@ public function setUp(): void
 
     public function testConstructorRaisesExceptionWhenMasterPidIsNotZero(): void
     {
+        /** @var SwooleHttpServer&MockObject $httpServer */
```

Comment:
Adding this resolves IDE completion

Reply:
Phpstorm already has autocomplete for this 🤔

Reply:
intelephense does not. :smile:

Reply:
Beware:

 * phpcs rules (especially `slevomat/*`) tent to replace `/** @var T&V $value */` with `assert($value instanceof T && $value instanceof V)`
 * `assert($value instanceof T && $value instanceof V)` is considered redundant by `vimeo/psalm`

Not a problem for now, but these hints will be removed in some non-far future, where inference can be obtained by psalm ;-)